docs/pages/chat-apps/list-stream-sync/list.mdx

```diff
@@ -93,4 +93,44 @@ The `isActive()` method determines whether the current user is still an active m
 - When a user is added to a group, `isActive()` returns `true` for that user
 - When a user is removed from a group, `isActive()` returns `false` for that user
 
-You can use a user's `isActive: true` value as a filter parameter when listing conversations. You can potentially have a separate section for "archived" or "inactive" conversations where you could use `isActive: false`.
+:::code-group
+
+```js [Browser]
+// Check if user is active in a specific conversation
+const isUserActive = await conversation.isActive();
+console.log('User is active:', isUserActive);
+
+// Filter active conversations
+const activeConversations = [];
+for (const conversation of allConversations) {
+  if (await conversation.isActive()) {
+    activeConversations.push(conversation);
+  }
+}
+```
+
+```js [Node]
+// Check if user is active in a specific conversation  
+const isUserActive = conversation.isActive;
+console.log('User is active:', isUserActive);
+
+// Filter active conversations (direct property access)
+const activeConversations = allConversations.filter(conversation => conversation.isActive);
+```
+
+```tsx [React Native]
```

Comment:
just noting that RN also has isActive() functions that works the same as js browser for conversation / group / dm.

Also you dont need to pass in false for all those parameters for conversations.list(), you can just call:

```typescript
const alixConvoCount1 = await alix.conversations.list({
  isActive: true // Only show active conversations
})
```